index.html

```diff
@@ -1381,6 +1381,47 @@
       </section>
     </section>
 
+    <section>
+      <h2>Verification</h2>
+
+      <p>
+This section describes a number of checks required to verify a claim.
+Some checks are essential for all verifiable claims, while some are applicable to only some claims.
+      </p>
+      <section>
+        <h3>Structural Validity</h3>
+        <p>
+          <ul>
+            <li>Document is valid JSON-LD.</li>
+            <li>Required properties are present. For example, for a Credential, <code>id</code>, <code>type</code>, and <code>claim</code> are required.</li>
+            <li>Property values match expectations described in the specification. For example, the document type for a verifiable claim must contain the class "Credential".</li>
+          </ul>
+        </p>
+      </section>
+      <section>
+        <h3>Entity Validity</h3>
+        <p>A number of checks </p>
+        <p>
+          <ul>
+            <li>The <code>issuer</code> id must match expectations. Likely, that means it is the id of a known and trusted <a>identity profile</a>.</li>
+            <li>The <a>claim id</a> must match expectations. Likely, that means it is the id of a known and trusted <a>identity profile</a> for the subject of the claim. If the entity that is subject of a claim has transmitted it to the inspector, the subject may be able to prove ownership of key identifyin properties such as email address(es) and public key(s).</li>
```

Comment:
'claim id', should probably be 'subject identifier' or 'subject of the claim'. We need to make sure this terminology exists in the terminology section (I'm working on that and will make sure we have whatever you put in here in the terminology section).